python/pyspark/sql/streaming.py

```diff
@@ -612,9 +616,12 @@ def load(self, path=None, format=None, schema=None, **options):
         :param schema: optional :class:`pyspark.sql.types.StructType` for the input schema.
         :param options: all other string options
 
-        >>> json_sdf = spark.readStream.format("json") \\
-        ...     .schema(sdf_schema) \\
-        ...     .load(tempfile.mkdtemp())
+        >>> import tempfile
+        >>> from pyspark.sql.types import StructType, StructField, StringType
+        >>> sdf_schema = StructType([StructField("data", StringType(), False)])
+        >>> json_sdf = (spark.readStream.format("json")
+        ...             .schema(sdf_schema)
+        ...             .load(tempfile.mkdtemp()))
```

Comment:
Strictly, I believe the removed syntax is encouraged as I guess from the other codes and documentation. We might better revert the syntax back if there is not a strong specific reason.